usecases/memwatch/monitor_test.go

```diff
@@ -159,37 +159,43 @@ func TestMappings(t *testing.T) {
 		mappingsLeft := getMaxMemoryMappings() - currentMappings
 		assert.InDelta(t, mappingsLeft, addMappings, 10) // other things can happen at the same time
 		path := t.TempDir()
+
+		limitReached := false
+
 		// use up available mappings
-		for i := 0; i < int(mappingsLeft)+5; i++ {
+		for i := 0; i < int(mappingsLeft)*2; i++ {
```

Comment:
**question (non-blocking):**  why 5 or 2 ?

Reply:
in the updated test it is only asserting that it hits the limit at some point, so I thought I better overrequest a lot (note that it is now *****2 and before it was **+**5)